daemon/daemon_unix.go

```diff
@@ -159,6 +159,21 @@ func getBlkioWeightDevices(config containertypes.Resources) ([]specs.WeightDevic
 	return blkioWeightDevices, nil
 }
 
+func parseAndSetSecurityOpt(daemon *Daemon, container *container.Container, config *containertypes.HostConfig) error {
```

Comment:
What's the purpose of this new function?
Isn't `parseSecurityOpt` already setting some values?